### ISSUE TITLE
No devices appearing in Home app

### ISSUE THREAD

**Issue by rogersmj** [home-assistant/homebridge-homeassistant#43]
I got homebridge and homebridge-homeassistant installed and running on my Raspberry Pi, and I was able to add it to my Home app on my iPhone, but none of my Home Assistant devices are appearing. I just see a "Default Room Homebridge" and there's nothing in it.

I looked through the docs but it says all the Home Assistant devices are just supposed to appear magically. I don't see any errors being logged in the terminal where I'm running Homebridge.

I have the following in my config file:

```
{
    "bridge": {
        "name": "Homebridge",
        "username": "CC:22:3D:E3:CE:30",
        "port": 51826,
        "pin": "xxx-xx-xxx"
    },


    "platforms": [
        {
                "platform": "HomeAssistant",
                "name": "HomeAssistant",
                "host": "http://192.168.1.117:8123",
                "password": "xxxxxxxxxx",
                "supported_types": ["light", "switch", "media_player", "scene"]
        }
    ]
}
```

Password x'd out for obvious reasons.

How can I debug this? Any suggestions?

**Comment by maddox** [comment]
When you started up Homebridge, did you see a list of your devices go by? Please provide the log output of Homebridge after it starts up.

**Comment by rogersmj** [comment]
No, nothing in the log other than it started up successfully and showed me the PIN.

I resolved this by ripping out node and reinstalling it. I was seeing hints that there was something messed up in my node installation (I think left over from an accidental duplicate node install I had awhile back). I destroyed all my node/nodejs binaries/libraries and started fresh. Now it works...mostly. Having a different issue where it says two devices are using the same ID. Will post a separate thread for that.
